RobotCode/src/main/java/frc/robot/subsystems/ShooterMountSubsystem.java

```diff
@@ -0,0 +1,79 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.core.motors.TeamSparkMAX;
+import frc.robot.constants.Ports;
+import java.util.Scanner;
+
+public class ShooterMountSubsystem extends SubsystemBase
+{
+
+	private TeamSparkMAX mainMotor, followMotor;
+
+	private double currentRotation;
```

Comment:
These doubles can be compressed into a single line (see line 11 for how to do this)